Source/ee/Vif.cpp

```diff
@@ -87,10 +87,13 @@ uint32 CVif::GetRegister(uint32 address)
 			result |= (0x10 << 24);
 		}
 		break;
+	case VIF0_ERR:
+	case VIF1_ERR:
+		result = m_ERR;
+		break;
 	case VIF0_MARK:
 	case VIF1_MARK:
 		result = m_MARK;
-		m_STAT.nMRK = 0;
```

Comment:
Did it help any game to remove that line? 

It does seem like the right thing to do according to documentation: MRK bit is supposed to be cleared when writing to MARK, not when reading. But I'm pretty sure I did that to fix a game... at the same time maybe the game in question also wrote to the MARK register.

Reply:
Sadly, no. This is just implemented against docs :(

Reply:
Fair enough. It probably a mistake on my end. Let's keep your change.